templeton.yaml.tmpl

```diff
@@ -7,6 +7,7 @@ amqp:
     {{ with $v := (key (printf "%s/amqp/exchange/name" $base)) }}name: {{ $v }}{{ end }}
     {{ with $v := (key (printf "%s/amqp/exchange/type" $base)) }}type: {{ $v }}{{ end }}
   {{- end }}
+  {{ with $v := (key (printf "%s/templeton/amqp-queue-prefix" $base)) }}queue_prefix: {{ $v }}{{ end }}
```

Comment:
(this ties into the defaults thing: if we're using the `with`, etc. here, then there should be a default set)